src/test/features/controls-spec.ts

```diff
@@ -426,9 +458,9 @@ suite('ModelViewerElementBase with ControlsMixin', () => {
 
                 canvas.blur();
 
+                const modelLoads = waitForEvent(element, 'load');
                 element.src = ASTRONAUT_GLB_PATH;
-
-                await waitForEvent(element, 'load');
+                await modelLoads;
```

Comment:
Is this just for code clarity, or does this affect flakiness?

Reply:
It was a hopeful attempt at the later... but I don't think it worked.